wolfcrypt/src/tfm.c

```diff
@@ -1739,7 +1739,7 @@ static int _fp_exptmod_nct(fp_int * G, fp_int * X, fp_int * P, fp_int * Y)
   if (M == NULL) {
      return FP_MEM;
   }
-  res = &M[1 << winsize];
+  res = &M[(word32)(1 << winsize)];
```

Comment:
Does using `1U` also silence this warning? Seems a bit cleaner than the cast.

Reply:
`1ULL`. Making the change.